protected/modules/_teacher/controllers/_admin/ModuleController.php

```diff
@@ -95,18 +97,18 @@ public function actionUpdate($id)
             $model->oldLogo = $model->module_img;
             $model->attributes = $_POST['Module'];
 
-            if (!empty($_FILES)) {
+            if (!empty($_FILES['Module']['name']['module_img'])) {
                 $imageName = array_shift($_FILES['Module']['name']);
-
+                $tmpName = array_shift($_FILES['Module']['tmp_name']);
                 if (!empty($imageName)) {
-                    $tmpName = array_shift($_FILES['Module']['tmp_name']);
                     if (!empty($imageName)) {
                         $model->logo = $_FILES['Module'];
                         if ($model->validate()) {
                             $model->save();
-
-                            if (!Avatar::updateModuleAvatar($imageName, $tmpName, $id, $model->oldLogo))
-                                throw new CDbException(400, 'Avatar not save');
+                            if($imageName && $tmpName) {
+                                if (!Avatar::updateModuleAvatar($imageName, $tmpName, $id, $model->oldLogo))
+                                    throw new CDbException(400, 'Avatar not save');
```

Comment:
Why you use CDbException for this? Is it ok here? Also can you correct message "Avatar was not saved"?